scripts/asamanager.sh

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env bash
+# Copyright (c) Microsoft. All rights reserved.
+# Note: Windows Bash doesn't support shebang extra params
+set -e
+
+if [[ "$DOCKER_TAG" == "" ]]; then
+    export DOCKER_TAG=testing
+fi
+
+DOCKER_IMAGE="azureiotpcs/asa-manager-dotnet:$DOCKER_TAG"
+DOCKER_PORT=9024
+DOCKER_NAME="asa-manager"
+DOCKER_NETWOK="integrationtests"
+
+APP_HOME="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && cd .. && pwd )/"
+cd $APP_HOME
+
+source "$APP_HOME/scripts/.functions.sh"
+
+check_env_variables()
```

Comment:
Since docker run command has already input environment variables from the env.list, is this step still necessary?

Reply:
yes because env.list doesn't set all variables--it pulls the ones it is checking for here from the machine